reference/sectionC_interpretation.inc

```diff
@@ -339,8 +339,8 @@ Interpretation of encapsulation
 
 .. _specC_interpretation_of_map_variables:
 
-Interpretation of map_variables
--------------------------------
+Interpretation of ``map_variables``
```

Comment:
This is an element too right?

Reply:
Yeah, I guess so.  In the beginning I was only adding "elements" to the titles to avoid having plurals in the monospace font (eg:  `variable`s or `variables`).  I guess I should just make them all long and elementy though.

Reply:
When editing the title, it would be good to add the corresponding number of `-`. I don't believe it's strictly needed, but it looks "better" when looking at the source code.

Reply:
Should the "Interpretation of mathematics" section now become "Interpretation of `math` elements"?

Reply:
I would say so.

Reply:
Both done in 906fe31

Reply:
Oops, my bad, I should have checked the commit!